scripts/yml/update_instructions.yaml

```diff
@@ -0,0 +1,3 @@
+openshift.image_pull_policy: Always
```

Comment:
IMO this is not needed, or maybe have it as another parameter for the installer script?

Reply:
it is also present in [mobile-core installer](https://github.com/aerogear/mobile-core/blob/0a4d41c6e7cfeec091b5574f76c944b392b9e392/installer/roles/ansible-service-broker-setup/defaults/main.yml#L10)